meshio/abaqus/_abaqus.py

```diff
@@ -165,6 +177,17 @@ def read_buffer(f):
                         cell_sets[name].append(cell_sets_element[set_name])
                     else:
                         raise ReadError(f"Unknown cell set '{set_name}'")
+        elif keyword == "INCLUDE":
+            # Get absolute path to referred input file
+            incl_ref = line.split("=")[-1].rstrip().replace("\\", "/")
```

Comment:
No idea what this all means. You can simply convert the string to a Path and do your business there, will be much easier.

Reply:
Ah, yes that did get messy. How about ```pathlib.Path(line.split("=")[-1])```?

Reply:
Better, but I still don't get what's up with the equality sign.

Reply:
Okay, well the "include" statement in abaqus is `*INCLUDE,INPUT=wInclude_bulk.inp`. I just split the line on the equal sign to create a list and use the last entry to get the path to the external input file. There are probably a bunch of alternative ways of doing this, but this was just the first that popped into my head :)

Reply:
No it's fine, I just didn't know the syntax. You can best add a code comment here with this example.

Reply:
Yup, will do.